images/twitter.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+    <div id= "twitterContainer">
+        <div class="left">
+          <div class="logoTwitter">
+            <img src="/asect/twitterlogo.webp" alt="Twitter logo" width="60px">
+          </div>
+          <div class="menuItem">
+         
+            <div class="menuLogo"><img src="/asect/home1.webp" alt="Home" width="30px"> </div>
```

Comment:
try wraping these img+text div in individual divs, that way you can hover over them individually instead of the whole section changing when you hover over one.